content/projects/gsoc/2020/project-ideas/machine-learning.adoc

```diff
@@ -22,4 +21,74 @@ links:
+4. Follow the Jenkins Plugin Tutorial [4] and create a simple Builder plug-in, that prints Hello World
+5. Modify the plug-in (which is also a Maven project) to call the code created in item 3
+6. Prove that we are able to call an IPython Kernel from a Jenkins job
+
+Extra steps could involve investigate and document for future developers (which is a good practice in Software Engineering) what would be required to use other Kernels (e.g. Perl, R, Groovy, etc), where the code would have to be modified, and perhaps also see if we could use Apache Zeppelin Interpreters [5] from Jenkins.
+
+=== References
+1. Models in Minutes not Months: Data Science as Microservices
+2. https://towardsdatascience.com/5-reasons-why-jupyter-notebooks-suck-4dc201e27086
+3. https://github.com/apache/zeppelin/blob/master/python/README.md
+4. https://jenkins.io/doc/developer/tutorial/
+5. https://zeppelin.apache.org/docs/0.7.0/manual/interpreters.html
+
+=== Links
+* Apply to Google Summer of Code
+ https://jenkins.io/projects/gsoc/
+* Discussion in the Developer mailing list: https://groups.google.com/forum/m/#!topic/jenkinsci-dev/VrBoOWVsZ-o
+* Creating a Jenkins Data Science Platform: http://biouno.org/2016/03/07/Creating-a-Jenkins-Data-Science-Platform.html
+
+=== Inspiration or Evaluation
+Other tools that researchers use for experiment tracking (should not be seen as a direct alternative! Some can be used together with Jenkins!)
+* https://colab.research.google.com/
+* https://neptune.ml/ ( ​$99/user/mo)
+* https://www.comet.ml/ ( ​$39/user/mo)
+* https://polyaxon.com/  ​ ​
+* https://github.com/ModelChimp/modelchimp  ​
+* https://github.com/datmo/datmo
+* https://github.com/IDSIA/sacred + https://github.com/vivekratnavel/omniboard / https://github.com/chovanecm/sacredboard
+* Airflow
+* studio.ml
+* https://medium.com/polyaxon
+* Version Control System for Machine Learning Projects https://dvc.org/
+
+=== TensorFlow
+* https://medium.com/@u39kun/managing-your-machine-learning-experiments-and-making-them-repeatable-in-tensorflow-pytorch-bc8043099dbd
+* https://github.com/tensorflow/tensorboard
+
+=== Jupyter-Zeppelin
+* https://www.dataquest.io/blog/jupyter-notebook-tutorial/
+* https://github.com/jupyter/jupyter/wiki/A-gallery-of-interesting-Jupyter-Notebooks
+* https://nextjournal.com/schmudde/how-to-version-control-jupyter
+* Existing Java integration with IPython Kernel (Jupyter), via Apache Zeppelin
+** https://github.com/apache/zeppelin/blob/master/docs/interpreter/python.md#ipython-support
+** https://github.com/apache/zeppelin/blob/master/python/README.md
+
+=== Example Jenkins Pipeline
+* An example pipeline launching Tensorboard from a Jenkins pipeline can be found [here](https://gist.github.com/imoutsatsos/256239cb2eb8a9a5932520e77601656b)
+
+=== Newbie-friendly issues
+There are currently no issues to reference but help can be requested via the gitter channel [here](jenkinsci/gsoc-machine-learning-project)
```

Comment:
Not a hyperlink, markdown is used instead of asciidoc

Reply:
I corrected this @oleg-nenashev